aggregations/csv/near_ecosystem_entities.csv

```diff
@@ -1,149 +1,152 @@
+marnotaur,Marnotaur,"AMM-based, cross-chain liquidity protocol for secure decentralized margin trading.",https://marnotaur.com,"defi, amm",building,,/img/marnotaurlogo.png,0,0,0
+webassembly-music,WebAssembly Music,Music stored on-chain as Non-Fungible Token.,https://petersalomonsen.com/nftmusic/2/,"app, nft",building,,/img/webassembly-music-logo.png,1,0,1
+hash-rush,Hash Rush,Online sci-fi/fantasy RTS set in the fictional Hermeian galaxy.,https://hashrush.com/,"app, nft, gaming",,,/img/hashrush.png,1,0,1
+npk-guild,NPK Guild,NEAR community based in Nigeria,,guild,,,/img/near-nigeria.png,0,1,0
+antimatter,AntiMatter,The gateway for DeFi derivatives & financial NFT’s.,https://antimatter.finance/,"app, defi, derivatives",building,,/img/antimatterlogo.png,1,0,0
+arterra,Arterra,The Digital Collectible Hub of Esports,https://arterra.co/,"app, nft, guild",,,/img/arterra.png,1,1,1
+skyward-finance,Skyward Finance,Enable fair token distribution and price discovery for projects built on NEAR Protocol.,https://skyward.finance/,"defi, app, guild",launched,skyward.near,/img/skywardfinancelogo.png,1,1,0
+skynet,Skynet,Decentralized Internet for a Free Future - host your content & build apps using decentralized storage,https://siasky.net/,tools,,,/img/skynet.png,0,0,0
+openrpc,OpenRPC,"The OpenRPC Specification defines a standard, programming language-agnostic interface description for JSON-RPC 2.0 APIs.",https://open-rpc.org/,infrastructure,building,,/img/openrpc-logo.png,0,0,0
+hackathon-guild,Hackathon Guild,Creating and supporting hackathons across the NEAR Ecosystem,,guild,,,,0,1,0
+nearpad,NearPad,Launchpad & DeFi hub for the NEAR ecosystem!,https://nearpad.io/,"app, defi",,,/img/nearpad.png,1,0,0
+trust-wallet,Trust Wallet,Fast and secure Ethereum wallet that’s easy to use.,https://trustwallet.com/,"infrastructure, wallet",launched,,/img/trustwallet.png,0,0,0
+autonomy-network,Autonomy Network,An off-the-shelf generalized automation solution powering the Web3 with on-chain conditional execution.,https://www.autonomynetwork.io/,infra,building,,/img/autonomylogo.png,0,0,0
+catalyst,Catalyst,Launch and participate in DAO-based communities quickly and easily with no code.,https://vitalpoint.ai/catalyst,"dao, app, education, social",mainnet,cdao.near,/img/catalyst.png,1,0,0
+nearverse-portal-guild,NEARverse Portal Guild,A guild that will focus on creating and curating information throughout the NEARVerse to keep the new portal relevant and updated.,https://gov.near.org/t/community-buidlers-guild-nearverse-portal-aka-wiki/4233/6,guild,,,/img/near-portal.png,0,1,0
+fuzzy-dice,Fuzzy Dice,Web3 Multidisciplinary Usability Guild,www.fu22y.com,guild,,,/img/dice.png,0,1,0
+quidli,Quidli,Micro-incentive rewards application for teams and individuals alike.,https://quid.li/,"app, ecosystem",,,/img/quidli.png,1,0,0
+near-india-guild,NEAR India Guild,NEAR Protocol's community based in India,,guild,,,/img/near-india.png,0,1,0
```

Comment:
nit: it's better to add the blank line at the end of the file. In this case, if you will add new lines at the end, you will not edit previous last line, it simplifies the review

Reply:
file removed